crime-commons-classes/src/main/java/uk/gov/justice/laa/crime/util/DateUtil.java

```diff
@@ -48,6 +54,18 @@ public static LocalDateTime convertDateToDateTime(LocalDate date) {
 
     public static ZonedDateTime toZonedDateTime(LocalDateTime localDateTime) {
         return (localDateTime == null) ? null :
-        ZonedDateTime.parse(localDateTime.atZone(ZoneId.systemDefault()).format(DateTimeFormatter.ISO_OFFSET_DATE_TIME));
+                ZonedDateTime.parse(localDateTime.atZone(ZoneId.systemDefault()).format(DateTimeFormatter.ISO_OFFSET_DATE_TIME));
+    }
+
+    public static LocalDateTime stringToLocalDateTime(String dateString, String dateFormat) {
```

Comment:
Optional changes:
I would consider returning `Optional<LocalDateTime>` such that when the dateString/dateFormat cannot be converted to a `LocalDateTime`, the calling class can decide how best to proceed.
Also, if ther date is unparsable, rather throw an exception, return an `Optional.empty()`.

Reply:
Good suggestion, but  I think it is better to throw an exception if date is un parsable, otherwise the calling class won't know about incorrect date.